src/components/LayoutLanding.vue

```diff
@@ -32,5 +35,7 @@ export default defineComponent({
 </script>
 
 <style scoped>
-
+.bg-white {
+    background-color: white;
+}
```

Comment:
I hadn't seen that, but since I had a difficult time finding the place that defined this style, I would argue it's better to have the style definition right there alongside the template code that uses it. There shouldn't be any clashes since the `style` section is `scoped` to only this component anyway. Let me know what you think.